conf/application.conf

```diff
@@ -140,8 +140,8 @@ microservice {
 mongodb {
     uri = "mongodb://localhost:27017/"${appName}
 
-    pension-scheme-event-reporting-cache {
-            maxSize = 512000
+    event-reporting-cache {
```

Comment:
Let's remove this event-reporting-cache {}, because all of the them are event-reporting cache. 

```
            event-reporting-data {
                name = event-reporting-data
                timeToLiveInDays = 28
            }
            
            should be enough, right ? 
```

Reply:
Change adopted in commit https://github.com/hmrc/pension-scheme-event-reporting/pull/24/commits/a14b180b05014b03092b096eae12a34d5fbcad29